GuessTheMovie/ImageStorage.h

```diff
@@ -58,8 +65,14 @@ struct ImageStorage
 	}
 
 	// TODO: implement this function
-	void create_missing_images(const Film& film_name, const std::string& path_to_save) 
-	{}
+	void create_missing_images(const Film& film_name, const std::string& path_to_save)
+	{
+		std::vector<Image> missing_images;
+		for (int i = 0; i < 4; ++i)
+		{
+			missing_images.push_back(dummy_image());
+		}
+	}
```

Comment:
What does this achieve? We should use @Sakuradaa's newly created structs inside this function, not this